lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/components/UstadHorizontalDivider.kt

```diff
@@ -0,0 +1,28 @@
+package com.ustadmobile.libuicompose.components
```

Comment:
This should be removed. Jetpack compose has a [HorizontalDivider](https://developer.android.com/develop/ui/compose/components/divider). We don't need to add our own